client/my-sites/email/email-providers-comparison/price-with-interval/style.scss

```diff
@@ -3,5 +3,5 @@
 }
 
 .price-with-interval__sale-price {
-	color: var( --studio-green-50 );
+	color: var( --studio-green-50 ) !important;
```

Comment:
I don't know what is uglier, but this works for me:

```
.email-provider-stacked-card__title-price-badge {
	.price-badge {
		.promo-card__price {
			.price__cost {
				.price-with-interval__sale-price {
					color: var( --studio-green-50 );
				}
			}
		}
	}
}
```

I would leave also the original rule, being
```
.price-with-interval__sale-price {
	color: var( --studio-green-50 );
}
```

Just in case we need to reuse this component out of a `price_cost` component

Reply:
That would be the old `Email Comparison` page. Thinking about it more, maybe we shouldn't update that page now (more on this in another comment). I'm going to revert to my original approach (i.e. without `!important`), which was very similar to what Daniel proposed.